backend/src/authentication/gamma.strategy.ts

```diff
@@ -0,0 +1,63 @@
+import Strategy, { Authority } from "./strategy.js";
+import passport from "passport";
+import { GammaUser } from "../models/gammaUser.js";
+
+const default_options = {
+  authorizationURL: "http://localhost:8081/api/oauth/authorize",
+  tokenURL: "http://localhost:8081/api/oauth/token",
+  profileURL: "http://localhost:8081/api/users/me",
+  clientID: "id",
+  clientSecret: "secret",
+  callbackURL: "http://localhost:3001/auth/account/callback",
+};
+
+const isAdmin = (authorities: Authority[], groups: String[]): boolean => {
+  if (process.env.MOCK && groups.includes("superadmin")) {
+    return true;
+  }
+  for (const i in authorities) {
+    if (authorities[i].authority == process.env.ADMIN_AUTHORITY || groups.includes("drawit")) {
+      return true;
+    }
+  }
+
+  return false;
+};
```

Comment:
Do we really have site admins? Or are these just going to be organizations?

Reply:
As of the new organization plan this is actually relevant as we will have a site admin to manage orgs. Groups will probably be digIT